src/components/SeriousRiddle.astro

```diff
@@ -8,16 +8,16 @@
 		<div id="complete" class="text-3xl"></div>
 		<div id="continue-click" class="vanishing mt-1/4"></div>
 	</div>
-	<div class="lg:min-ml-1/4 lg:min-mr-1/4 lg:mx-auto mt-8 w-fit">
+	<div class="lg:min-ml-1/4 lg:min-mr-1/4 mx-auto mt-8 w-fit">
 		<div id="title"></div>
 		<ul id="alternatives" class="mt-4 <lg:mt-8 w-fit mx-auto">
-			<li class="alternative glow cursor-pointer w-fit <lg:border-white <lg:border-2 <lg:mt-2 <lg:p-1"></li>
-			<li class="alternative glow cursor-pointer w-fit <lg:border-white <lg:border-2 <lg:mt-2 <lg:p-1"></li>
-			<li class="alternative glow cursor-pointer w-fit <lg:border-white <lg:border-2 <lg:mt-2 <lg:p-1"></li>
-			<li class="alternative glow cursor-pointer w-fit <lg:border-white <lg:border-2 <lg:mt-2 <lg:p-1"></li>
+			<li class="alternative glow cursor-pointer w-fit <lg:text-left <lg:mt-1 <lg:p-1"></li>
+			<li class="alternative glow cursor-pointer w-fit <lg:text-left <lg:mt-1 <lg:p-1"></li>
+			<li class="alternative glow cursor-pointer w-fit <lg:text-left <lg:mt-1 <lg:p-1"></li>
+			<li class="alternative glow cursor-pointer w-fit <lg:text-left <lg:mt-1 <lg:p-1"></li>
 		</ul>
 		<div id="result" class="hidden mt-4 ml-auto mr-auto <lg:mt-8"></div>
-		<a id="continue" class="glow hidden mt-4 cursor-pointer ml-auto mr-auto w-fit <lg:border-2 <lg:p-1">Next &raquo;</a>
+		<a id="continue" class="glow hidden mt-4 cursor-pointer ml-auto mr-auto w-fit <lg:p-1">Next &raquo;</a>
 	</div>
```

Comment:
@gianlucastefanoni can you fix the merge conflict regarding this file?
looks like there was some work regarding the Quiz on the Trex branch

Reply:
in particular the merge conflict is this one
```
<<<<<<< trex-integration-branch
	<div class="lg:min-ml-1/4 lg:min-mr-1/4 mx-auto mt-8 w-fit">
=======
	<div id="quizContainer" class="lg:min-ml-1/4 lg:min-mr-1/4 mx-auto mt-8 w-fit">
>>>>>>> dev
```
but the other changes are probably also in conflict with the quiz branch